<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Generic3DPlotSettings</class>
 <widget class="QWidget" name="Generic3DPlotSettings">
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="cameraGridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="zoomLabel">
       <property name="text">
        <string>Zoom</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="zoomSpinBox">
       <property name="toolTip">
        <string>Sets the zoom percentage</string>
       </property>
       <property name="suffix">
        <string>%</string>
       </property>
       <property name="minimum">
        <number>25</number>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="zoomResetButton">
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Resets the zoom to 100%</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="cameraLabel">
       <property name="text">
        <string>Camera</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="cameraResetButton">
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Resets the zoom to 100%</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="cameraAzDoubleSpinBox">
       <property name="wrapping">
        <bool>true</bool>
       </property>
       <property name="prefix">
        <string>Az=</string>
       </property>
       <property name="suffix">
        <string>°</string>
       </property>
       <property name="minimum">
        <double>-179.990000000000009</double>
       </property>
       <property name="maximum">
        <double>180.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QDoubleSpinBox" name="cameraElDoubleSpinBox">
       <property name="wrapping">
        <bool>true</bool>
       </property>
       <property name="prefix">
        <string>El=</string>
       </property>
       <property name="suffix">
        <string>°</string>
       </property>
       <property name="minimum">
        <double>-90.000000000000000</double>
       </property>
       <property name="maximum">
        <double>90.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="mShowLocator">
     <property name="title">
      <string>Locator</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="1">
       <widget class="QLineEdit" name="mValueLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>Select a pattern</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="mAzimuthLabel">
        <property name="text">
         <string>Azimuth:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="mAzimuthLineEdit">
        <property name="text">
         <string>0.00</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="mElevationlabel">
        <property name="text">
         <string>Elevation:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="mElevationLineEdit">
        <property name="text">
         <string>0.00</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="mValueLabel">
        <property name="text">
         <string>Value:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
